tests/aws_iot_devicedefender_client_test.c

```diff
@@ -56,28 +73,28 @@ static void s_mqtt_on_connection_complete(
     enum aws_mqtt_connect_return_code return_code,
     bool session_present,
     void *userdata) {
-
     (void)connection;
     (void)error_code;
     (void)return_code;
     (void)session_present;
-    struct connection_args *connection_args = (struct connection_args *)userdata;
+    struct connection_args *args = (struct connection_args *)userdata;
 
     AWS_FATAL_ASSERT(error_code == AWS_ERROR_SUCCESS);
     AWS_FATAL_ASSERT(return_code == AWS_MQTT_CONNECT_ACCEPTED);
     AWS_FATAL_ASSERT(session_present == false);
 
-    printf("Client connected...");
-
-    defender_task = aws_iotdevice_defender_v1_report_task(connection_args->allocator, &connection_args->task_config);
+    AWS_FATAL_ASSERT(
+        AWS_OP_SUCCESS ==
+        aws_iotdevice_defender_task_create(&defender_task, args->task_config, connection, args->defender_event_loop));
+    // now register signal handler
+    signal(SIGINT, sigint_handler);
```

Comment:
Why are we handling `SIGINT` explicitly?

Reply:
It probably had a purpose for someone in a prior phase. Removing.